projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item-list/cart-item-list.component.ts

```diff
@@ -28,9 +31,6 @@ export class CartItemListComponent implements OnInit {
     });
   }
 
-  @Input()
```

Comment:
Removing/changing the input of existing component is considered a breaking change.